nomad/jobs/nginx.nomad.hcl

```diff
@@ -0,0 +1,74 @@
+job "nginx" {
+
+  group "nginx" {
+    count = 1
+
+    network {
+      port "http" {
+        static = 8080
+      }
+    }
+
+    service {
+      name = "nginx"
+      port = "http"
+    }
+
+    task "nginx" {
+      driver = "docker"
+
+      config {
+        image = "nginx"
+        network_mode = "host"
+
+        ports = ["http"]
+
+        volumes = [
+          "local:/etc/nginx/conf.d",
+        ]
+      }
+
+      template {
+        data = <<EOF
+{{ range services }}
+
+{{ $host := "localhost" }}
```

Comment:
Why is $host being set to localhost here and then to .ServiceMeta.host later?

Reply:
`.ServiceMeta.host` is available for each service in the inner loop, not here. So I'm initializing it to `localhost` and updating it in inner loop.